src/patternfly/components/Table/table.scss

```diff
@@ -30,53 +29,100 @@
-  --pf-c-table-cell--first-last-child--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-table-cell--first-last-child--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-table--cell--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-table--cell--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-table--cell--Color: var(--pf-global--Color--100);
+  --pf-c-table--cell--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-table--cell--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-table--cell--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-table--cell--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-table--cell--first-last-child--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-table--cell--first-last-child--PaddingRight: var(--pf-global--spacer--lg);
+
+  // Default cell variables
+  --pf-c-table--cell--MinWidth: 0;
+  --pf-c-table--cell--MaxWidth: none;
+  --pf-c-table--cell--Width: auto;
+  --pf-c-table--cell--Overflow: visible;
+  --pf-c-table--cell--TextOverflow: clip;
+  --pf-c-table--cell--WhiteSpace: normal;
+  --pf-c-table--cell--WordBreak: normal;
+
+  // Truncate
+  --pf-c-table--m-truncate--cell--MinWidth: 10ch;
+  --pf-c-table--m-truncate--cell--MaxWidth: 1px;
+  --pf-c-table--m-truncate--cell--Overflow: hidden;
+  --pf-c-table--m-truncate--cell--TextOverflow: ellipsis;
+  --pf-c-table--m-truncate--cell--WhiteSpace: nowrap;
+
+  // Wrap
+  --pf-c-table--m-wrap--cell--MinWidth: 0;
+  --pf-c-table--m-wrap--cell--MaxWidth: none;
+  --pf-c-table--m-wrap--cell--Overflow: visible;
+  --pf-c-table--m-wrap--cell--TextOverflow: clip;
+  --pf-c-table--m-wrap--cell--WhiteSpace: normal;
+
+  // No wrap
+  --pf-c-table--m-no-wrap--cell--MinWidth: 0;
+  --pf-c-table--m-no-wrap--cell--MaxWidth: none;
+  --pf-c-table--m-no-wrap--cell--WhiteSpace: nowrap;
+
+  // Fit content
+  --pf-c-table--m-fit-content--cell--MinWidth: fit-content;
+  --pf-c-table--m-fit-content--cell--MaxWidth: fit-content;
+  --pf-c-table--m-fit-content--cell--Width: 1%;
+  --pf-c-table--m-fit-content--cell--WhiteSpace: nowrap;
+
+  // Hidden visible
+  --pf-c-table--cell--hidden-visible--Display: table-cell;
+  // stylelint-enable
 
   // ============================================================ //
   // End non-conformant variables
   // ============================================================ //
 
+  // Text
+  --pf-c-table__text--MaxWidth: auto;
+
   // Table cell - toggle
   --pf-c-table__toggle--c-button--MarginTop: calc(#{pf-size-prem(6px)} * -1);
   --pf-c-table__toggle--c-button__toggle-icon--Transform: rotate(270deg);
   --pf-c-table__toggle--c-button__toggle-icon--Transition: .2s ease-in 0s;
   --pf-c-table__toggle--c-button--m-expanded__toggle-icon--Transform: rotate(360deg);
+
+  // Button
+  --pf-c-table__button--BackgroundColor: transparent;
+  --pf-c-table__button--Color: var(--pf-global--Color--100);
+  --pf-c-table__button--hover--Color: var(--pf-global--Color--100);
```

Comment:
do we need all of these if the color is the same for default and hover/active/focus

Reply:
Yes, I think so. @mcoker wdyt?

Reply:
If we anticipate that this could change, I think so. If we set all of the states with 1 variable, then decide to introduce a new var for one of the states, since the initial var that was used no longer controls that state, it's technically a breaking change.